NICE.Identity.Authorisation.WebAPI/DataModels/Organisation.cs

```diff
@@ -7,22 +8,22 @@ public class Organisation
         public Organisation()
         {
         }
-
         public Organisation(int organisationId, string name)
         {
             OrganisationId = organisationId;
             Name = name;
         }
-
-        public  int OrganisationId { get; set; }
+        public int OrganisationId { get; set; }
         public string Name { get; set; }
+        public DateTime? DateAdded { get; set; } = DateTime.UtcNow;
         public ICollection<OrganisationRole> OrganisationRoles { get; set; }
         public ICollection<Job> Jobs { get; set; }
 
-        public void UpdateFromApiModel(ApiModels.Organisation organisation)
+        public void UpdateFromApiModel(ApiModels.Organisation organisation, bool create)
```

Comment:
If the migration script is changed to give a default DateAdded can we do away with the create flag?